docs/integrations/spark.rst

```diff
@@ -0,0 +1,122 @@
+Lance ❤️ Spark
+--------------------
+
+Lance can be used as a third party datasource of `<https://spark.apache.org/docs/latest/sql-data-sources.html>`_
+
+.. warning::
+    This feature is experimental and the APIs may change in the future.
+
+Build from source code
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+    git clone https://github.com/lancedb/lance.git
+    cd lance/java
+    mvn clean package -DskipTests -Drust.release.build=true
+
+After building the code, the spark related jars are under path :class:`lance/java/spark/target/jars/`
+
+.. code-block:: shell
+
+	arrow-c-data-15.0.0.jar
+	arrow-dataset-15.0.0.jar
+	jar-jni-1.1.1.jar
+	lance-core-0.25.0-SNAPSHOT.jar
+	lance-spark-0.25.0-SNAPSHOT.jar
+
+
+
+Download the pre-build jars
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you did not want to get jars from source, you can download these five jars from maven repo.
+
+.. code-block:: bash
+
+	wget https://repo1.maven.org/maven2/com/lancedb/lance-core/0.23.0/lance-core-0.23.0.jar
```

Comment:
Can we use maven to declare dependency? Those steps look excessive.

Reply:
Does a user of Spark usually use Maven?

If that's not possible, I wonder if we can offer a script that can be run off Github in a single line, similar how you can install `rustup` with just:

```bash
curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```